hbp_knowledge/anatabine/paris2013.bel

```diff
@@ -146,115 +160,149 @@ anatabine (Mann–Whitney U=0, Z=-2.309, P=0.021) and a significant
 acetylcholine receptor subtype (Suzuki et al., 2006)"
+SET Confidence = "Medium"
+SET Species = "9606"
 SET Cell = "microglial cell"
 a(MESH:Microglia) -> a(MESH:"alpha7 Nicotinic Acetylcholine Receptor")
 UNSET Cell
+UNSET Species
+UNSET Confidence
 
+#not sure of information provide: was LPS given with anatabine?
 SET Evidence = "An increased
 STAT3 (Mann–Whitney U=0, Z=-2.309, P=0.021) and p65 NFkB
 phosphorylation (Mann–Whitney U=0, Z=-2.309, p=0.021) was
 observed in LPS treated microglia (Fig. 3)"
+#SET Confidence
 SET Cell = "microglial cell"
 a(CHEBI:lipopolysaccharide) -> p(HGNC:STAT3,pmod(Ph))
 a(CHEBI:lipopolysaccharide) -> p(HGNC:RELA,pmod(Ph))
 UNSET Cell
+#UNSET Confidence
 
 SET Evidence = "We observed that anatabine significantly suppressed the stimulation of p65 NFkB and
 STAT3 phosphorylation by LPS in microglia (Fig. 3)"
+#SET Confidence = "High"
 SET Cell = "microglial cell"
 a(CHEBI:lipopolysaccharide) -> p(HGNC:STAT3,pmod(Ph))
 a(CHEBI:Anatabine) -| p(HGNC:STAT3,pmod(Ph))
 a(CHEBI:lipopolysaccharide) -> p(HGNC:RELA,pmod(Ph))
 a(CHEBI:Anatabine) -| p(HGNC:RELA,pmod(Ph))
 UNSET Cell
+#UNSET Confidence
 
 SET Evidence = "Kruskal–Wallis test revealed that doses of anatabine significantly suppressed both
 LPS induced STAT3 phosphorylation (H=15.658, df=4, P=0.005)
 and p65 NFkB phosphorylation (H=14.150, df=4, P=0.007) in human microglia"
+SET Confidence = "Medium"
+SET Species = "9606"
 SET Cell = "microglial cell"
 a(CHEBI:Anatabine) -| p(HGNC:STAT3,pmod(Ph))
 a(CHEBI:Anatabine) -| p(HGNC:RELA,pmod(Ph))
 UNSET Cell
+UNSET Species
+UNSET Confidence
 
 SET Evidence = "A significant inhibition of STAT3 phosphorylation
 was observed with 10 mg/ml of anatabine (Mann–Whitney U=1,
 Z=-2.021, P=0.043), with 100 mg/ml of anatabine (Mann–Whitney
 U=0, Z=-2.309, P=0.021), with 300 mg/ml of anatabine
 (Mann–Whitney U=0, Z=-2.309, P=0.021) and with 600 mg/ml
 of anatabine (Mann–Whitney U=0, Z=-2.309, P=0.021)"
+SET Confidence = "High"
 SET Cell = "microglial cell"
 # Dosage: 10 mg/ml, 100 mg/ml, 300 mg/ml and 600 mg/ml of anatabine
 a(CHEBI:Anatabine) -| p(HGNC:STAT3,pmod(Ph))
 UNSET Cell
+UNSET Confidence
 
 SET Evidence = "Similarly, LPS induced p65 NFkB phosphorylation in microglia was
 significantly inhibited with 10 mg/ml of anatabine (Mann–Whitney
 U=0, Z=-2.309, P=0.021), with 100 mg/ml of anatabine (Mann–
 Whitney U=0, Z=-2.309, P=0.021), with 300 mg/ml of anatabine
 (Mann–Whitney U=0, Z=-2.309, P=0.021)"
+SET Confidence = "High"
 SET Cell = "microglial cell"
 # Dosage: 10 mg/ml, 100 mg/ml and 300 mg/ml of anatabine
 a(CHEBI:lipopolysaccharide) -> p(HGNC:RELA,pmod(Ph))
 a(CHEBI:Anatabine) -| p(HGNC:RELA,pmod(Ph))
 UNSET Cell
+UNSET Confidence
 
 SET Evidence = "TNFa significantly stimulated
 STAT3 (Mann–Whitney U=0, Z=-2.309, P=0.021) and p65
 NFkB phosphorylation (Mann–Whitney U=0, Z=-2.309, P=0.021)"
-SET CellLine = "HEK293"
+SET Confidence = "High"
+#No cell line mentioned in evidence
```

Comment:
Maybe it was mentioned in the sentence before, I don't remember...
Just skip the CellLine annotation here